### PR TITLE
Replace supervisor with tsx --watch

### PR DESCRIPTION
**Pull request by junlarsen**
Works just like before with reloading of properties. See demo GIF

Peek 2024-12-11 12-10

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,15 +97,15 @@ run:  ## Runs the site like it runs in production
 
 .PHONY: dev
 dev: prereqs ## Runs the site as a developer; including live reload support and installation of git hooks
-	NODE_OPTIONS="$(TS_NODE_ARGS) $(NODE_ARGS)" ./node_modules/.bin/supervisor -w app.ts,lib,etc/config,static/tsconfig.json -e 'js|ts|node|properties|yaml' -n exit --exec $(NODE) -- ./app.ts $(EXTRA_ARGS)
```
Comment (junlarsen) [inline Makefile:97]: TSX automatically watches any file in the import graph, so we shouldn't have to list out everything anymore, only the "extra" files.

Comment (mattgodbolt) [reply Makefile:97]: "any file in the import graph" -> including configuration files?

Comment (mattgodbolt) [reply Makefile:97]: oh!!! I see above the `etc/config` :)

**Review by mattgodbolt — APPROVED** [review]
Hurrah!

**Comment by mattgodbolt** [comment]
Thanks so much @junlarsen !

**Pull request merged**